annotation/populate_database.py

```diff
@@ -95,6 +102,40 @@ def _try_create_generation(gen_text, system, prompt, decoding_strategy):
 @click.option('--generations_path', help='JSON file containing generations.')
 @click.option('--version', help='Version number.')
 def populate_db(generations_path, version):
+    # populate pre-set feedback options
+    grammar_option = _try_create_feedback_option(
```

Comment:
Can you put this all in a helper function? It might also make more sense to put the feedback options in a CSV or json, and then just read in that file here.

Reply:
yes! done with csv now